content/source/docs/cloud/registry/add.html.md

```diff
@@ -1,50 +1,54 @@
 ---
 layout: "cloud"
-page_title: "Adding Public Modules - Private Module Registry - Terraform Cloud and Terraform Enterprise"
+page_title: "Adding Public Providers and Modules - Private Registry - Terraform Cloud and Terraform Enterprise"
 description: |-
-  Learn to add public modules from the public Terraform Registry to your organization's private module registry.
+  Add providers and modules from the public Terraform Registry to your organization's private registry.
 ---
 
 [vcs]: ../vcs/index.html
 
-# Adding Public Modules to the Terraform Cloud Private Module Registry
+# Adding Public Providers and Modules to the Private Registry
 
-> **Hands-on:** Try the [Add Public Modules to your Private Module Registry](https://learn.hashicorp.com/tutorials/terraform/module-private-registry-add?in=terraform/modules&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
+You can add providers and modules from the public [Terraform Registry](/docs/registry/index.html) to your Terraform Cloud private registry. This lets you clearly designate which public providers and modules are recommended for the organization and makes their supporting documentation and examples centrally accessible.
 
-You can add modules from the public [Terraform Registry](/docs/registry/index.html) to your Terraform Cloud private module registry. Once added, Terraform Cloud automatically synchronizes public modules with their source on the Terraform Registry. The private module registry handles downloads and controls access with Terraform Cloud API tokens, so consumers don't need access to the module's source repository, even when they run Terraform from the command line.
+~> **Note:** Terraform Enterprise does not support public providers or modules.
```

Comment:
It does! To use this feature in Terraform Enterprise, you must enable access to both registry.terraform.io and https://yy0ffni7mf-dsn.algolia.net .

Reply:
I thought I saw that that TFE was coming slightly later - maybe in the next release in December - so it won't quite be available yet when these docs are first up. Can you just confirm that on launch this will be available in both TFC and TFE? Once you do I will fix this and add in the extra bits for TFE :-)

Reply:
okay we discussed offline, and for now the decision is to keep the note as is. Will update again for the December TFE release. Thank you :-)